src/patternfly/demos/Card/card-demo-toolbar.hbs

```diff
@@ -0,0 +1,45 @@
+{{#> toolbar toolbar--id="toolbar-card-example"}}
+  {{#> dropdown dropdown--modifier="pf-c-toolbar__bulk-select" dropdown--IsBulkSelect="true" dropdown--IsSplitButton="true" dropdown-toggle--type="div" dropdown-toggle--modifier="pf-m-split-button" dropdown-menu--toggle-id="toolbar-simple-split-button-dropdown-simple-example-button"}}
+    {{> dropdown-toggle-check dropdown-toggle-check--id=(concat toolbar--id '-split-button-dropdown-simple-example-check') aria-label="Select all"}}
+    {{> dropdown-toggle-button dropdown--IsToggleButton="true" dropdown-toggle-button--id=(concat toolbar--id '-split-button-dropdown-simple-example-button')  aria-label="Select"}}
+  {{/dropdown}}
+  {{#> button button--modifier="pf-m-plain pf-c-toolbar__filter-toggle" aria-label="Filter results"}}
+    {{> button-icon button-icon--type="filter"}}
+  {{/button}}
+  {{#> toolbar-filter}}
+    {{#> select id=(concat toolbar--id '-select-checkbox') select--IsCheckboxSelect="true"}}
+      <i class="fas fa-filter" aria-hidden="true"></i>
+      Filter
+    {{/select}}
+  {{/toolbar-filter}}
+  {{#> toolbar-action-group}}
+    {{#> button button--modifier="pf-m-primary"}}
+      Create a project
+    {{/button}}
```

Comment:
This just disappears. Should we display the kebab so it's still available?

Reply:
Yeah good idea, wasn't in design, but makes sense with the current toolbar